server/db/user.js

```diff
@@ -28,10 +35,13 @@ const User = db.define('user', {
     allowNull: false,
```

Comment:
Thoughts on allowing address to be null; Thinking about sign-up and user creation, If I had to enter an address for anything I probably wouldn't sign up. We could do validation on PURCHASE/CHECKOUT for an address, but may not be necessary here

Reply:
Address isn't set as "allowNull"-- I think you were looking at "isAdmin" ?

Reply:
^what kim said, this is resolved